### PR TITLE
Adding discord link

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ F3D also contains the libf3d, a simple library to render meshes, with C++, Pytho
 
 See the [gallery](doc/GALLERY.md) for more images, take a look at the [changelog](doc/CHANGELOG.md) or go to the [releases page](https://github.com/f3d-app/f3d/releases) to download F3D!
 
+If you need any help or want to discuss with other F3D users and developers, head over to our [discord](https://discord.gg/ZYHzNeEdX5).
```
Comment (Meakk) [inline README.md:28]: Can we add a badge on the homepage too?

Comment (Meakk) [reply README.md:28]: Is the link valid permanently?

Comment (mwestphal) [reply README.md:28]: Good point for the badge !

Yes, link is never expiring.

**Review by Meakk — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mwestphal) [reply README.md:28]: a

**Comment by mwestphal** [comment]
@snoyer : here is the discord link: https://discord.gg/ZYHzNeEdX5 :)

**Pull request merged**